assets/js/src/actions.js

```diff
@@ -72,6 +72,8 @@ addUniqueAction(
 	trackAddToCart
 );
 
+eventListener( `${ EVENT_PREFIX }-add_to_cart`, trackAddToCart );
+
```

Comment:
Is there a reason to use jQuery here? I think we can reduce the size of dependency and bloat by using native JS
```js
document.body.addEventListener(
    `${ EVENT_PREFIX }-add_to_cart`,
    event =>  trackAddToCart( {product: event.detail } )
);
// ...
document.body.dispatchEvent(
    new CustomEvent( 
        `google_analytics-add_to_cart`, 
        {detail: {foo: 'bar'}}
    )
);
     

```